virttest/utils_libvirtd.py

```diff
@@ -29,24 +30,51 @@ class Libvirtd(object):
     Class to manage libvirtd service on host or guest.
     """
 
-    def __init__(self, session=None):
+    def __init__(self, service_name=None, session=None):
```

Comment:
Usually when we add a new parameter, it is better to append it which may avoid of existing codes' conflicts for the parameters.

Reply:
I've fixed parameter order issue in tp-libvirt 2916 and here.

Reply:
OK if you would like to fix in all occurrences.